uisoup/win_soup/mouse.py

```diff
@@ -179,7 +179,7 @@ def click(self, x, y, button_name=LEFT_BUTTON):
             self._compose_mouse_event(button_name, press=True, release=True),
             0, 0, 0, 0)
 
-    def double_click(self, x, y, button_name=LEFT_BUTTON):
+    def double_click(self, x, y, button_name=LEFT_BUTTON, dbl_time=0.5):
```

Comment:
Also you need to add this parameter to https://github.com/yandex-qatools/UISoup/blob/master/uisoup/interfaces/i_mouse.py and to https://github.com/yandex-qatools/UISoup/blob/master/uisoup/mac_soup/mouse.py